docs/conf.py

```diff
@@ -85,7 +85,7 @@
     {"file":"about/release-notes", "os":["linux"]},
 ]
 
-exclude_patterns = ['temp']
+exclude_patterns = ['temp', 'release/versions.md']
```

Comment:
I'm not sure if we want to exclude this. There's no explicit link in the repo, but the header includes it via rocm-docs-core (top right Version List)

Reply:
Removed now. I will ask the team about rocm-docs-core and will return to this.